rust/operator-binary/src/superset_controller.rs

```diff
@@ -265,13 +336,59 @@ fn build_node_rolegroup_service(
     })
 }
 
+/// The rolegroup [`ConfigMap`] configures the rolegroup based on the configuration given by the administrator
+fn build_rolegroup_config_map(
+    rolegroup: &RoleGroupRef<SupersetCluster>,
+    superset: &SupersetCluster,
+    rolegroup_config: &HashMap<PropertyNameKind, BTreeMap<String, String>>,
+    authentication_class: &Option<AuthenticationClass>,
+) -> Result<ConfigMap> {
+    let mut cm_conf_data = BTreeMap::new(); // filename -> filecontent
+
+    for property_name_kind in rolegroup_config.keys() {
+        match property_name_kind {
+            PropertyNameKind::File(file_name) if file_name == SUPERSET_CONFIG => {
+                let superset_config = compute_superset_config(authentication_class);
+                cm_conf_data.insert(SUPERSET_CONFIG.to_string(), superset_config);
```

Comment:
Why not just `config_map_builder.add_data` this directly?

Reply:
i copied it from the druid operator. But i can simply it